src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    newRobot[robot[key]] = key;
+  }
+
+  if (Object.keys(robot).length !== Object.keys(newRobot).length) {
+    return null;
+  }
```

Comment:
you can check the property existence inside a loop
```suggestion
  for (const key in robot) {
    if (newRobot[robot[key]]) {
      return null;
    }

    newRobot[robot[key]] = key;
  }
```